pipelines/jobs/pipeline_job_template.groovy

```diff
@@ -62,16 +62,16 @@ pipelineJob("${BUILD_FOLDER}/${JOB_NAME}") {
                     // Do not inherit permissions from global configuration
                     nonInheriting()
                 }
-                permissions(['hudson.model.Item.Build:AdoptOpenJDK*build', 'hudson.model.Item.Build:AdoptOpenJDK*build-triage',
-                'hudson.model.Item.Cancel:AdoptOpenJDK*build', 'hudson.model.Item.Cancel:AdoptOpenJDK*build-triage',
-                'hudson.model.Item.Configure:AdoptOpenJDK*build', 'hudson.model.Item.Configure:AdoptOpenJDK*build-triage',
-                'hudson.model.Item.Read:AdoptOpenJDK*build', 'hudson.model.Item.Read:AdoptOpenJDK*build-triage',
+                permissions(['GROUP:hudson.model.Item.Build:AdoptOpenJDK*build', 'GROUP:hudson.model.Item.Build:AdoptOpenJDK*build-triage',
+                'GROUP:hudson.model.Item.Cancel:AdoptOpenJDK*build', 'GROUP:hudson.model.Item.Cancel:AdoptOpenJDK*build-triage',
+                'GROUP:hudson.model.Item.Configure:AdoptOpenJDK*build', 'GROUP:hudson.model.Item.Configure:AdoptOpenJDK*build-triage',
+                'GROUP:hudson.model.Item.Read:AdoptOpenJDK*build', 'GROUP:hudson.model.Item.Read:AdoptOpenJDK*build-triage',
                 // eclipse-temurin-bot needs read access for TRSS
-                'hudson.model.Item.Read:eclipse-temurin-bot',
+                'GROUP:hudson.model.Item.Read:eclipse-temurin-bot',
                 // eclipse-temurin-compliance bot needs read access for https://ci.eclipse.org/temurin-compliance
-                'hudson.model.Item.Read:eclipse-temurin-compliance-bot',
-                'hudson.model.Item.Workspace:AdoptOpenJDK*build', 'hudson.model.Item.Workspace:AdoptOpenJDK*build-triage',
-                'hudson.model.Run.Update:AdoptOpenJDK*build', 'hudson.model.Run.Update:AdoptOpenJDK*build-triage'])
+                'GROUP:hudson.model.Item.Read:eclipse-temurin-compliance-bot',
```

Comment:
```suggestion
                'USER:hudson.model.Item.Read:eclipse-temurin-compliance-bot',
```

Reply:
Yep that was clearly wrong and why I wanted a good set of second eyeballs on it - thanks :-)